modules/exploits/multi/http/confluence_widget_connector.rb

```diff
@@ -0,0 +1,494 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'json'
```

Comment:
Neither of these should need to be required.

Reply:
Done.